lib/puppet/http/service.rb

```diff
@@ -1,7 +1,8 @@
 class Puppet::HTTP::Service
   attr_reader :url
 
-  SERVICE_NAMES = [:ca, :report].freeze
+  SERVICE_NAMES = [:ca, :puppet, :report].freeze
```

Comment:
nit: should this change be in this commit?

Reply:
Updated

Reply:
It looks like you haven't pushed up the change yet